jerry-core/jerryscript.h

```diff
@@ -322,9 +329,19 @@ jerry_value_t jerry_get_object_keys (const jerry_value_t obj_val);
 jerry_value_t jerry_get_prototype (const jerry_value_t obj_val);
 jerry_value_t jerry_set_prototype (const jerry_value_t obj_val, const jerry_value_t proto_obj_val);
 
+__attribute__((deprecated))
 bool jerry_get_object_native_handle (const jerry_value_t obj_val, uintptr_t *out_handle_p);
+__attribute__((deprecated))
 void jerry_set_object_native_handle (const jerry_value_t obj_val, uintptr_t handle_p,
                                      jerry_object_free_callback_t freecb_p);
+
+bool jerry_get_object_native_handle_ext (const jerry_value_t obj_val,
```

Comment:
I agree with @jiangzidong that with the word "native" it may be a bit clearer as to what it does.
So I'd +1 `jerry_set_object_native_pointer` and `jerry_get_object_native_pointer`